proto/utxorpc/v1alpha/query/query.proto

```diff
@@ -11,6 +11,8 @@ import "utxorpc/v1alpha/cardano/cardano.proto";
 message ChainPoint {
   uint64 slot = 1; // Slot number.
   bytes hash = 2; // Block hash.
+  uint64 height = 3; // Block height
+  uint64 timestamp = 4; // Block ms timestamp
```

Comment:
Here, I put `timestamp` in millseconds even though Cardano only uses seconds for two reasons:
1. Some chains have milliseconds as well (and `ChainPoint` could be used for other chains)
2. Some networks like Midgard could have sub-second blocks, and maybe even Cardano could have this with Leios in the future. I'd rather not pointlessly limit ourselves to second-resolution then regret it later